examples/bri-3/src/bri/zkp/agents/proofStorage.agent.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { BlockchainService } from '../../../../blockchain/blockchain.service';
+
+import { Proof } from '../models/proof';
+
+@Injectable()
+export class ProofStorageAgent extends BlockchainService {
```

Comment:
not sure why storage agent should extend blockchain service? i am still looking into this a first time, but you should inject blockchain service in constructor

Reply:
+1

Reply:
+1